docs/api-testing/project-access.md

```diff
@@ -85,7 +85,7 @@ To enable email notifications:
    <br/><img src={useBaseUrl('img/api-testing/notifications.png')} alt="Dashboard Pic 1.5"/>
 
 :::tip
-Use the Fact component in your API tests to control the behavior of email notifications. This can be useful if you're testing in multiple environments. For more information, see [Fact](/api-testing/composer/other-components/#fact).
+Use the Fact component in your API tests to control the behavior of email notifications. This can be useful if you're testing in multiple environments. For more information, see [Fact](/api-testing/composer/other-components/#fact) and [Use Cases](/api-testing/use-cases/fact/).
```

Comment:
```suggestion
Use the Fact component in your API tests to control the behavior of email notifications. It can be beneficial when you're testing in multiple environments. For more information, see [Fact](/api-testing/composer/other-components/#fact) and [Use Cases](/api-testing/use-cases/fact/).
```